chromeos.kernelci.org

```diff
@@ -6,6 +6,17 @@
 
 set -e
 
+# Trap function for errors
+crash() {
+    local exit_code="$?"
```

Comment:
`local` is a Bash keyword.  I guess you could directly compare `$?` with 0 and not use a variable?

Reply:
Thanks for noticing it, i forgot why i defined it - it is to keep correct exit variable in case of error. Updated now.

It is also defined in BSD shells, busybox ash, dash, etc.
Also according to debian policy manual:
```
Scripts may assume that /bin/sh implements the POSIX.1-201[7](https://www.debian.org/doc/debian-policy/ch-files.html#id22) Shell Command Language 7 plus the following additional features not mandated by POSIX.1-2017.. [8](https://www.debian.org/doc/debian-policy/ch-files.html#id23)
...

    local to create a scoped variable must be supported, including listing multiple variables in a single local command and assigning a value to a variable at the same time as localizing it. local may or may not preserve the variable value from an outer scope if no assignment is present. Uses such as:
```

Reply:
OK, I've hit cases in the past where it was not supported. So maybe we can make the script explicitly call `/bin/dash` (or bash...)